docs/source/configuration/reference.rst

```diff
@@ -187,6 +187,9 @@ Common configuration is a subdictionary that is a value of ``ext`` key in
     ``out`` and ``rewrite`` prompts (refer to IPython documentation for more 
     details) while ``in`` prompt is the default.
 
+    For the wm ext, it is a dictionary ``{output : theme_name}`` that maps 
+    the ``xrandr`` output names to the local theme to substitute on that output.
```

Comment:
“themes to use” (plural (to match “names”) and “use”), not “theme to substitute” (singular and “substitute”).